predeval/predeval.py

```diff
@@ -28,6 +28,10 @@
 import random
 import itertools
 import csv
+import json
+import datetime
+import time
+import progress
```

Comment:
The import and usage of `progress`

> On Jun 6, 2016, at 2:51 PM, Sam Steingold <notifications@github.com> wrote:
> 
> In predeval/predeval.py:
> 
> > @@ -28,6 +28,10 @@
> >  import random
> >  import itertools
> >  import csv
> > +import json
> > +import datetime
> > +import time
> > +import progress
> what do you want to remove?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>